common/frame.go

```diff
@@ -1505,8 +1505,23 @@ func (f *Frame) SetContent(html string, opts goja.Value) {
 
 // SetInputFiles is not implemented.
 func (f *Frame) SetInputFiles(selector string, files goja.Value, opts goja.Value) {
-	k6ext.Panic(f.ctx, "Frame.setInputFiles(selector, files, opts) has not been implemented yet")
-	// TODO: needs slowMo
+	f.log.Debugf("Frame:SetInputFiles", "fid:%s furl:%q sel:%q", f.ID(), f.URL(), selector)
+
+	popts := NewFrameSetInputFilesOptions(f.defaultTimeout())
+	if err := popts.Parse(f.ctx, opts); err != nil {
+		k6ext.Panic(f.ctx, "parsing setInputFiles options: %w", err)
```

Comment:
It would be great if we could return an error from this function instead of panicking.